src/gui_common/CustomRichTextLabel.cs

```diff
@@ -632,4 +644,58 @@ private void OnMetaClicked(object meta)
             }
         }
     }
+
+    private void OnMetaHoverStarted(object meta)
+    {
+        if (!EnableTooltipsForMetaTags || speciesPreviewTooltipScene == null)
+            return;
+
+        if (meta is string metaString && metaString.StartsWith("species", StringComparison.Ordinal))
+        {
+            var speciesCode = GUICommon.ConvertFromSafeMetaTag(metaString.Split("species:")[1]);
```

Comment:
This split could be avoided here by just getting a substring that skips the "species:" amount of characters. I think that would be a slightly less garbage generating / faster operation.